subsys/bluetooth/mesh/friend.c

```diff
@@ -1161,6 +1186,8 @@ static void friend_timeout(struct k_work *work)
 		return;
 	}
 
+	update_overwrite(sys_slist_peek_head(&frnd->queue) != NULL);
```

Comment:
should be `update_overwrite(frnd->last, sys_slist_peek_head(&frnd->queue) != NULL);`